theme/less/main.less

```diff
@@ -42,49 +56,35 @@ header h1 a:hover
 }
 
 main
-{
-  margin: 0px auto;
-  padding: 15px;
-  max-width: 1000px;
+{ 
+  grid-area: main;
+  padding: 15px 0;
 }
 
+img
+{
+  max-width: 100%;
+}
 
 footer
 {
+  grid-area: footer;
   border-top: 1px solid #d6d6d6;
   padding: 40px;
-  line-height: 10px;
+  line-height: 15px;
   text-align: center;
   font-size: 15px;
 }
 
-#breadcrumb
-{
-  font-size: 0.92rem;
-  color: #999;
-  background-color: #f1f1f1;
-  border-radius: 4px;
-  border: 1px solid #e9e9e9;
-  text-shadow: 1px 1px 0px #fff;
-}
-
-#breadcrumb a
+@media screen and (max-width: 770px)
 {
-  color: #797979;
-  text-shadow: none;
-}
-
-#breadcrumb li:not([aria-current='page']):after
-{
-  content: '»';
-  margin: 0 6px;
-}
-
-#breadcrumb ol
-{
-  display: flex;
-  list-style: none;
-  padding: 5px 5px 5px 10px;
-  margin: 0;
-}
-
+  body > div
+  {
+    padding: 20px 30px;
```

Comment:
Detail: Seems like we are already specifying this style outside of `@media` query, I guess it's safe to remove it.